src/asciidoc/docs/develop/portExistingCode.adoc

```diff
@@ -214,25 +269,45 @@ That's because you will allow users to customize those sounds.
 . Press *Ctrl+S* to save the file, and then return to the *Visual* view.
 +
 You should now see two additional pickers that allow a user to choose sounds for your game.
-. Open the `frontend/index.js` file, and press *Ctrl+F* to open the finder tool.
-. Enter `.wav` to search for the wav files that the game is using.
+. Open the `frontend/index.js` file, and add the following code below the `upgradeBGColor` function.
```

Comment:
There are a few changes here to make this code work properly. You can check out https://withkoji.com/code/seane/asteroids-ir-test

Notably:

Add a `loadAudio` function, and be sure to call that and `upgradeBGColor` in the `onValueChanged` handler.

Make sure you are referencing things inside the `gameOptions` key